cognite_toolkit/_cdf_tk/utils/cdf.py

```diff
@@ -12,39 +12,6 @@
 from cognite_toolkit._cdf_tk.tk_warnings import MediumSeverityWarning
 
 
-def retrieve_view_ancestors(client: ToolkitClient, parents: list[ViewId], cache: dict[ViewId, View]) -> list[View]:
-    """Retrieves all ancestors of a view.
-
-    This will mutate the cache that is passed in, and return a list of views that are the ancestors of the views in the parents list.
-
-    Args:
-        client: The Cognite client to use for the requests
-        parents: The parents of the view to retrieve all ancestors for
-        cache: The cache to store the views in
-    """
```

Comment:
Also not necessary any more, are the motivation for looking up ancestors was to remove their properties.